client/verta/verta/monitoring/profiler.py

```diff
@@ -165,7 +165,7 @@ def profile_column(self, df, column):
         dict
             A dictionary from summary data names to DiscreteHistogram instances.
         """
-        content = df[column].value_counts()
+        content = df[column].value_counts(dropna=True)
```

Comment:
`True` is [already the default](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.value_counts.html), but this makes what's happening more explicit.